src/sentry/search/snuba/backend.py

```diff
@@ -122,6 +123,30 @@ def inbox_filter(inbox, projects):
     return query
 
 
+def owner_filter(owner, projects):
+    from sentry.models import Team
+
+    organization_id = projects[0].organization_id
+    project_ids = [p.id for p in projects]
+    if isinstance(owner, Team):
+        return Q(
+            id__in=GroupOwner.objects.filter(
+                team=owner, project_id__in=project_ids, organization_id=organization_id
+            )
+            .values_list("group_id", flat=True)
+            .distinct()
+        )
+
+    else:
```

Comment:
You don't need this else as your if block returns. Should this case be wrapped with `if isinstance(owner, User)`?

Reply:
I don't know what I'd do in the case that the owner isn't a User or a Team. Raise an error? Could I just let django raise an error when tying to assign a non-User to a User?

Removed the else

Reply:
You could raise an InvalidSearchQuery error.

Reply:
Done and done. Thanks.